server/src/main/java/org/elasticsearch/enrollment/EnrollmentSettings.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.enrollment;
+
+import org.elasticsearch.common.settings.Setting;
+
+public class EnrollmentSettings {
```

Comment:
Do we need a new class of settings for this single setting ?

Reply:
It doesn't fit very well to any other classes, so I leaving it here for now, but don't have a strong opinion

Reply:
Why does it have to be in a class though ? For what is worth, I've revisited the approach after discussions with Tim and it would suffice for this to be a node specific static setting. See updated design doc alsi